ANMN/LTSP/TSaggregator/aggregated_timeseries.py

```diff
@@ -111,12 +173,12 @@ def write_netCDF_aggfile(aggDataset, ncout_filename):
                                              'calendar': 'gregorian'},
```

Comment:
You've hard-coded 'units' and 'calendar' again here. These should come from the attributes of the TIME variable (you can read them from `aggDataset`).
By the way `agg_dataset` would be more consistent with our code conventions (and you also called the same argument in other functions `nc`...)

Reply:
This is a weird behaviour of the `to_netcdf` xarray method that overrides the units you set as attribute for the TIME variable. It set the units and calendar to 
```
TIME:units = "seconds since 2014-12-16 08:00:00.000003328" ;
TIME:calendar = "proleptic_gregorian" ;
```
So you need to hard code just before writing as a encoding dictionary and pass it as argument.

Reply:
What I mean is you can just say
```python
    encoding = {'TIME':                     {'_FillValue': False,
                                             'units': aggDataset.TIME.units,
￼	                                     'calendar': aggDataset.TIME.calendar},
```

Reply:
This is still an issue of the xarray. When it reads the netCDF file it automatically convert `TIME` to a `datetime64[ns]` object and don't keep the units/calendar attribute. see https://github.com/pydata/xarray/issues/2565.
To keep the units/cal attr in the output file I need to code it manually.

Off course I can force the non-decoding of `TIME` in the reading process (`decode_times=False`),   but it will require more changes in the code. Also getting the units/cal attr from the (first) file we will be assuming that those values are the same for all the files. I'm trying to avoid this approach even though here it should be a safe assumption.

Anyway, those values are a IMOS netCDF convention.

Reply:
solution found!.